### PR TITLE
Move to batch-based directory structure

### PR DESCRIPTION
**Pull request by lgruen**

### CODE REVIEW

File: scripts/move_uploaded_crams.py [hunk 1]
```diff
@@ -5,6 +5,11 @@
 import subprocess
 
 output = os.getenv('OUTPUT')
-assert output and output.startswith('gs://cpg-tob-wgs-archive/')
+assert output and output.startswith('gs://cpg-tob-wgs-archive/cram/batch')
```
Comment (vladsavelyev) [inline scripts/move_uploaded_crams.py:8]: Should it be `batch` or `batch0`?

Comment (lgruen) [reply scripts/move_uploaded_crams.py:8]: Good spotting :). This doesn't contain the `0` so we can run it for future batches too :)

Comment (vladsavelyev) [reply scripts/move_uploaded_crams.py:8]: Ah, gotcha. So in the command line, you would set it to `gs://cpg-tob-wgs-main/gvcf/batch0/`. Same for `move_uploaded_gvcfs.py`.

**Review by vladsavelyev — APPROVED** [review]

File: scripts/move_uploaded_crams.py (hunk 1 above)
Comment (lgruen) [reply scripts/move_uploaded_crams.py:8]: That's exactly right.

**Pull request merged**